### PR TITLE
Prepare 2.9.2 patch release

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Prepare 2.9.2 patch release

## Release Notes

Milestone: 2.9.2

Changelog: N/A

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]

**Review by traeok — DISMISSED** [review]

**Comment by JillieBeanSim** [comment]
@zFernand0 should we go ahead and announce the keytar migration plan in ZE README for the marketplace?

**Comment by zFernand0** [comment]
> @zFernand0 should we go ahead and announce the keytar migration plan in ZE README for the marketplace? If so, we can add in this PR
> 
> I could also add a v3 pre-release note like I did in [wiki](https://github.com/zowe/vscode-extension-for-zowe/wiki) to the README

I'm cool with announcing the keytar replacement plans in the readme.md (so that it makes its way to the marketplace).

**Comment by JillieBeanSim** [comment]
@zFernand0 added the warnings/notes, let me know if we should reword or anything
Screen Shot 2023-07-31 at 1 25 50 PM

File: packages/zowe-explorer/README.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 [![codecov](https://codecov.io/gh/zowe/vscode-extension-for-zowe/branch/main/graph/badge.svg)](https://codecov.io/gh/zowe/vscode-extension-for-zowe)
 [![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://slack.openmainframeproject.org/)
 
+> ## :rotating_light: With the removal of keytar shim from VS Code we will be replacing `node-keytar` with `keytar-rs` in our v2.10.0 release to avoid user interuptions
```
Comment (zFernand0) [inline packages/zowe-explorer/README.md:8]: `interuptions -> interruptions`
`keytar-rs -> @zowe/secrets-for-zowe-sdk`

**Review by zFernand0 — DISMISSED** [review]
left a small comment.
LGTM though 😋

File: packages/zowe-explorer/README.md (hunk 1 above)
Comment (JillieBeanSim) [reply packages/zowe-explorer/README.md]: haha don't approve with my misspellings lol
thanks, updates pushed.

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**